pom.xml

```diff
@@ -146,7 +98,8 @@
                 <scope>provided</scope>
             </dependency>
 
-            <!-- OSGi Dependencies -->
+            <!-- OSGi deprecated Dependencies -->
```

Comment:
I believe we should use this version as an opportunity to remove Felix dependencies as the separate projects begin using this version of `aem-bom`.

Reply:
good idea

Reply:
I removed the Felix dependencies from bom. I also refactored cru project removing Felix and using the new OSGi annotations.